### PR TITLE
Fix: profile stats modal

### LINKED ISSUE #1299: The Topics tab displays a white screen if the user has no Interests or Topics.
**Describe the bug**
When the user has no Interests or Topics, the Topics tab displays a white screen.

**To Reproduce**
1. Go to the Profile UI of the user.
2. Click the Interest icon to open the Topics tab.
3. Observe the empty Topics tab.

**Expected behavior**
A photo and message should display that the user has no topics yet.

**Screenshots**

NoTopicsMobile

Desktop/Laptop:
NoTopics

Toshiba Dynabook
OS: Windows 10 pro
Browser: Google Chrome
Version 93.0.4577.82
Browser: Firefox Version
Version 92.0

Smartphone:

Device: Sony Xperia 1
OS: 11
Browser Google Chrome
Version 77.3865.103

Device: Huawei P20
OS: 10
Browser Google Chrome
Version 93.0.4577.82
Browser MetaMask
Version 3.3.0

Device: iPhone XR
OS: 15
Browser Mozilla Firefox
Version 19.1
Browser MetaMask
Version 3.3.0
Browser Safari
Version 15
Browser Google Chrome
Version 77.3865.103

Wallet Used:
Metamask

EP(ACP)

cc: @ggolcher

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- use ethAddress as fallback for new profiles
- handle edge case in profile interest

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/hooks/src/use-profile.new.ts [hunk 1]
```diff
@@ -145,6 +145,11 @@ const getInterests = async (pubKey: string) => {
   try {
     const res = await lastValueFrom(sdk.api.profile.getInterests(pubKey));
 
+    // return, if empty
+    if (!res.data.getInterests.length) {
```
Comment (kenshyx) [inline ui/hooks/src/use-profile.new.ts]: this is patched in #1302 , you can remove the changes from use-profile

**Review by kenshyx — APPROVED** [review]
